controllers/component_controller_test.go

```diff
@@ -136,12 +137,10 @@ var _ = Describe("Component controller", func() {
 			}, timeout, interval).Should(BeTrue())
 
 			Eventually(func() bool {
-				fmt.Println(createdHasComp.ResourceVersion)
```

Comment:
Deleted this because it was priting undesired output to during the tests' run, I figured it was left here by accident.